youtube_dl/extractor/thisvid.py

```diff
@@ -0,0 +1,200 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import re
+import itertools
+
+from .common import InfoExtractor
+from ..compat import (
+    compat_urlparse,
+)
+from ..utils import (
+    clean_html,
+    get_element_by_class,
+    int_or_none,
+    merge_dicts,
+    urljoin,
```

Comment:
Restore redirect metadata pt.3.
```suggestion
    url_or_none,
    urljoin,
```